xla/pjrt/gpu/tfrt/BUILD

```diff
@@ -105,7 +105,14 @@ cc_library(
         "@tsl//tsl/profiler/lib:connected_traceme",
         "@tsl//tsl/profiler/lib:context_types_hdrs",
         "@tsl//tsl/profiler/lib:traceme",
-    ],
+    ] + if_cuda([
+        # keep sorted
+        "@local_config_cuda//cuda:cuda_headers",
+    ]) + if_rocm([
+        # keep sorted
+        "@local_config_rocm//rocm:rocm_headers",
+        "@local_config_rocm//rocm:rocm_config",
+    ]),
```

Comment:
if 0.6.0 doesn't have this commit https://github.com/ROCm/xla/commit/c4e87e83bc03031a38253908cb7116543b8e5bff , I don't think you need to add this change